conf/base.config

```diff
@@ -23,43 +23,43 @@ process {
   withLabel:'sc_tiny'{
       cpus = { check_max( 1, 'cpus' ) }
       memory = { check_max( 1.GB * task.attempt, 'memory' ) }
-      time = { check_max( 2.h * task.attempt, 'time' ) }
+      time = { check_max( 24.h * task.attempt, 'time' ) }
```

Comment:
Shouldn't tiny jobs not also be less than 24hours? e.g. lets say 3 for tiny, 6 for small, 8 for medium and 16 for large?

Reply:
My fear here is that some clusters actually send you to the end of the queue if you ask for more hours than you actually need - so it shouldn't be set to too much more than the initial process will most likely need.

Reply:
Tiny is actually referring to the resources rather than time (at least in the eager concept).

Hmm ok. That's tricky then. I originally honestly felt 2.h was enough but #545 made me realise this could occur when people run their own data. Maybe 4.h for everything? Half a day I think would be pretty sold.